### ISSUE TITLE
Retire this repository?

### ISSUE THREAD

**Issue by tarcieri** [socketry/socketry-async#3]
Now that https://github.com/socketry/async exists, what should become of this repo? Should we get rid of it?

cc @ioquatix

**Comment by ioquatix** [comment]
I thought briefly about this.

My gut feeling was initially to leave it here. It might make sense in the future to implement socketry-async whereby you can write `Socketry::Async::Reactor.new` and when you wrap sockets, you get instances of `Socketry` wrappers. But, this is not something I have time for nor would be interested in maintaining, simply because it's not of use to me right now and I think the vast majority of the issues you are addressing in `Socketry` are outside the scope of what `async` needs, what `async-dns` needs.

Don't get me wrong, I really like what `Socketry` does, and it's really useful to read the code. I just don't think it's the right level of abstraction for what I wanted/needed.

Additionally, I liked the logo and I wanted to think about how we can use it.  `Socketry::Async` is not correct, but perhaps we can name things with the cool socket iconography, and call it `Socketry/async`, `Socketry/nio4r` and so on. What do you think?

In summary, if you don't think this repository has a place going forward, I'd support your decision to retire it. I'm not going to maintain it or develop it as it's not in-line with my needs.

**Comment by tarcieri** [comment]
I don't really have any desire to do anything with it at this point, so I think it'd be best to delete it and avoid the associated confusion of having two similarly named repos, one of which is defunct.

**Comment by tarcieri** [comment]
Nuking it. If you want it again, I'll have a local copy for now...
